R/process_sc_all_care_home.R

```diff
@@ -85,47 +96,63 @@ process_sc_all_care_home <- function(
       -"min_ch_provider",
       -"max_ch_provider"
     ) %>%
-    # tidy up ch_provider using 6 when disagreeing values
     tidyr::fill(.data[["ch_provider"]], .direction = "downup") %>%
```

Comment:
Do we need this line? when running this, we change mismatching min/max cases to 6 on lines 89-93. There also shouldnt be any NA values as this was corrected on line 70. Im not sure what the fill is doing as my understanding is that we have already done this on lines 89-93.

Reply:
I think this is a leftover from the code that was there before. Need to look at it when more awake